apis/virtualKubelet/v1/namespacemap_types.go

```diff
@@ -0,0 +1,62 @@
+/*
+
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// NamespaceMapSpec defines the desired state of NamespaceMap
+type NamespaceMapSpec struct {
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "make" to regenerate code after modifying this file
+
```

Comment:
Is it correct the spec is empty? I feel it pretty strange

Reply:
At the moment we have nothing to implement here, so I left it empty... Should I remove it ?

Reply:
I don't think you can delete the spec entirely. I was just raising a concern that I felt strange to have a CRD with no spec and a status, but if it comes from a previous design phase and the @liqotech/maintainers agreed that is fine to me

Reply:
Thank you very much for the advise, we will consider again it.

Reply:
I agree it is pretty strange. For the time being, we don't have any spec in this API, but most likely we'll add some fields in the future. Since this API represents the current mapping between local and remote namespaces, we used only the status subresource.